# +(-)limit까지 운전
#limit -> 0: -limit, 1: +limit
int32 speed
int32 limit